mozilla-mobile/android-components/components/feature/addons/src/main/res/values/strings.xml

```diff
@@ -92,6 +92,17 @@
     <string name="mozac_feature_addons_rating_content_description">%1$.02f/5</string>
     <!-- This is the title of page where all the add-ons are listed-->
     <string name="mozac_feature_addons_addons">Add-ons</string>
+    <!-- The label of the allow button, this will be shown to the user when an addons needs new permissions, with the button the user will indicate that wants to accept the new permissions and update the addon-->
+    <string name="mozac_feature_addons_updater_notification_allow_button">Allow</string>
+    <!-- The label of the deny button on a notification, this will be shown to the user when an addons needs new permissions. Indicates the user denies the new permissions and prevent the addon from be updated-->
+    <string name="mozac_feature_addons_updater_notification_deny_button">Deny</string>
+    <!-- The tile of the notification, this will be shown to the user when an addons needs new permissions, to be updated. %1$s is the name of the addons-->
+    <string name="mozac_feature_addons_updater_notification_title">%1$s has a new update</string>
+    <!-- The content of the notification, this will be shown when an addons needs new permissions, to be updated. %1$d is the amount of new permissions-->
+    <string name="mozac_feature_addons_updater_notification_content" tools:ignore="PluralsCandidate">%1$d new permissions are required</string>
+    <!-- Name of the "notification channel" used for displaying a notification for updating an addon. See https://developer.android.com/training/notify-user/channels -->
+    <string name="mozac_feature_addons_updater_notification_channel">Addon updates</string>
```

Comment:
Noticed while reviewing translations: add-on is mispelled all over the place here (without an hyphen, "an addons").

Reply:
Thanks  for the catch, I'll file an issue for this.

Reply:
@Amejia481: Seems like I missed this issue. Can you help update? This should be consistent throughout products

Reply:
This should be good enough to do w/out ID changes?

Reply:
I think so.
Sorry, for the trouble, I will cover that work on https://github.com/mozilla-mobile/android-components/issues/5564

Reply:
Yes, but we'll need to fix en-GB and en-CA on our side ([see translations](https://transvision.mozfr.org/string/?entity=android_l10n/mozilla-mobile/android-components/components/feature/addons/src/main/res/values/strings.xml:mozac_feature_addons_updater_notification_channel&repo=android_l10n)).

Reply:
Then I only have to change `strings.xml `on AC?

Reply:
Or do you prefer that I change IDs?

Reply:
Correct. Only correct the XML file in android-components.

Note that only one string uses "addon", but there are a lot of comments with the wrong spelling, and in case of "an addons" an extra typo.

Reply:
No need to change the ID, we want to keep the existing translations.

Reply:
Thanks for the clarification 👍

Reply:
When you have a chance, please take a look and let me know, if there is some that I should change.
https://github.com/mozilla-mobile/android-components/pull/5566

Reply:
> Yes, but we'll need to fix en-GB and en-CA on our side ([see translations](https://transvision.mozfr.org/string/?entity=android_l10n/mozilla-mobile/android-components/components/feature/addons/src/main/res/values/strings.xml:mozac_feature_addons_updater_notification_channel&repo=android_l10n)).

I fixed en-GB in Pontoon, but we don't have en-CA. @flodolo Am I missing something? I think that's the only instance here.

Reply:
> I fixed en-GB in Pontoon, but we don't have en-CA. @flodolo Am I missing something? I think that's the only instance here.

No, somehow I saw en-CA in that list…